provisioner.go

```diff
@@ -39,17 +39,29 @@ func newDefaultProvisioner(l astikit.StdLogger) (dp *defaultProvisioner) {
 		},
 	})
 	dp = &defaultProvisioner{l: astikit.AdaptStdLogger(l)}
-	dp.moverAstilectron = func(ctx context.Context, p Paths) (err error) {
+	dp.moverAstilectron = func(ctx context.Context, p Paths) (closeFunc func() error, err error) {
 		if err = Download(ctx, dp.l, d, p.AstilectronDownloadSrc(), p.AstilectronDownloadDst()); err != nil {
-			return fmt.Errorf("downloading %s into %s failed: %w", p.AstilectronDownloadSrc(), p.AstilectronDownloadDst(), err)
+			return nil, fmt.Errorf("downloading %s into %s failed: %w", p.AstilectronDownloadSrc(), p.AstilectronDownloadDst(), err)
 		}
-		return
+		return func() (err error) {
+			if err = os.Remove(p.AstilectronDownloadDst()); err != nil {
+				return fmt.Errorf("removing %s failed: %w", p.AstilectronDownloadDst(), err)
+			}
+			dp.l.Debugf("Remove download file: %s", p.AstilectronDownloadDst())
+			return nil
+		}, err
 	}
-	dp.moverElectron = func(ctx context.Context, p Paths) (err error) {
+	dp.moverElectron = func(ctx context.Context, p Paths) (closeFunc func() error, err error) {
 		if err = Download(ctx, dp.l, d, p.ElectronDownloadSrc(), p.ElectronDownloadDst()); err != nil {
-			return fmt.Errorf("downloading %s into %s failed: %w", p.ElectronDownloadSrc(), p.ElectronDownloadDst(), err)
+			return nil, fmt.Errorf("downloading %s into %s failed: %w", p.ElectronDownloadSrc(), p.ElectronDownloadDst(), err)
 		}
-		return
+		return func() (err error) {
+			if err = os.Remove(p.ElectronDownloadDst()); err != nil {
+				return fmt.Errorf("removing %s failed: %w", p.ElectronDownloadDst(), err)
+			}
+			dp.l.Debugf("Remove download file: %s", p.ElectronDownloadDst())
```

Comment:
Adding a debug log for this action makes sense. I would however 

1) place it before `os.Remove()`
2) replace the text with `removing %s`

Reply:
done. https://github.com/asticode/go-astilectron/pull/358/commits/3481a4e396ea2fa4cef5282d957edcc4dd50505b#diff-752145c98ffd12d3a2b292517f8cf139ca3bcaa6512dc88dd7d8628372fd5df4R59-L62